<!DOCTYPE html>
<html lang="" xml:lang="">
  <head>
    <title>Genome-wide identification, structural analysis and expression profiling of pathogenesis-related protein 1 (PR-1) genes in soybean (Glycine max)</title>
    <meta charset="utf-8" />
    <meta name="author" content="Fabricio Almeida-Silva @almeidasilvaf" />
    <meta name="date" content="2021-10-15" />
    <script src="libs/header-attrs-2.11/header-attrs.js"></script>
    <link href="libs/remark-css-0.0.1/default.css" rel="stylesheet" />
    <script src="libs/xaringanExtra_fit-screen-0.2.6/fit-screen.js"></script>
    <link href="libs/panelset-0.2.6/panelset.css" rel="stylesheet" />
    <script src="libs/panelset-0.2.6/panelset.js"></script>
    <link rel="stylesheet" href="assets/venanciolab-fonts.css" type="text/css" />
    <link rel="stylesheet" href="assets/venanciolab.css" type="text/css" />
  </head>
  <body>
    <textarea id="source">
class: center, middle, inverse, title-slide

# Genome-wide identification, structural analysis and expression profiling of pathogenesis-related protein 1 (PR-1) genes in soybean (<em>Glycine max</em>)
### Fabricio Almeida-Silva <span class="citation">@almeidasilvaf</span>
### UENF / Brazil
### October 15, 2021

---








background-image: url(https://github.com/almeidasilvaf/GCN_GWAS_fungi/blob/main/figs/intro4.png?raw=true)
background-position: 95% 50%
background-size: 45%

## Introduction

&lt;br /&gt;

.footnote[Song *et al.*, 2020]

.pull-left[
Arms race in plant defense:

- PAMP-triggered immunity (PTI)

- Effector-triggered immunity (ETI)

Several associated genes, e.g.:

1. Recognition

2. Signal transduction

3. ROS production and oxidative burst

4. Physical barriers

5. .bgp[Pathogenesis-related proteins]

]

---

## Pathogenesis-related (PR) proteins

.footnote[Moosa *et al.*, 2017]

&lt;br /&gt;
PR proteins are classified in 17 families based on their sequence similarities, enzymatic activities and structural properties.

&lt;img src="https://www.researchgate.net/profile/Shahbaz-Sahi/publication/315476424/figure/tbl1/AS:613911694942262@1523379362944/Classification-of-PR-proteins.png" width="890%" style="display: block; margin: auto;" /&gt;

---

## PR-1

&lt;br /&gt;
PR-1-encoding genes are present in all plant taxa.

--

First identified in tobacco leaves infected with TMV: 2% of the proteins, suggesting a role in plant defense.

--

Widely neglected PR family.

--

Stress-responsive, but they also have endogenous functions.

- PR or PR-like?

--

Plants overexpressing PR-1 genes have increased resistance against fungi, oomycetes and bacteria.

--

Recent identification of a conserved CAP domain (cysteine-rich secretory protein, antigen 5, pathogenesis-related 1) in all PR-1.

---

## The CAP domain (PF00188)

&lt;br /&gt;
Binding and sequestration of sterols in pathogens' membranes.

- further evidence: more effective against sterol auxotrophs.

--

Typically folds into 4 `\(\alpha\)`-helices and 4 `\(\beta\)`-strands stabilized by disulfide bonds.

--

Last 11 residues can be cleaved and release a CAP-derived (CAPE) peptide.

--

CAPE peptide does not induce expression of WRKY TFs: independent signaling pathway.

---

## The research problem

&lt;br /&gt;
- Unexplored PR family in plants. Never fully characterized in soybean.

- Are PR-1 genes promising targets for biotechnological applications in soybean?

---

## Goals

&lt;br /&gt;
- Identify and characterize soybean genes encoding PR-1 genes at the .blue[sequence], .dark-orange[structure], and .green[expression] levels to fill the following gaps:

1. .blue[How many phylogenetically distinct groups are there?]

--

1. .blue[How did they emerge?]

--

1. .dark-orange[What are the physicochemical properties of soybean PR-1 proteins?]

--

1. .dark-orange[Are primary and tertiary structures conserved?]

--

1. .dark-orange[Do gene structures reflect the phylogeny?]

--

1. .green[How are PR-1 genes regulated?]

--

1. .green[Which genes are constitutively expressed and which are stress-specific?]

--

1. .green[Which stress conditions can induce PR-1 gene expression?]

---

## Identification of *GmPR-1* genes

&lt;br /&gt;
- .bgg[Method:] BLASTp search using *A. thaliana* PR-1 genes as queries (E-value: 1e-10).
- 24 *GmPR-1* genes, all of which contain the CAP domain.

---

background-image: url(figs/pr1/tabela_pr1.png)
background-size: contain

.bgg[Method:] &lt;br /&gt;
 R/Biostrings &lt;br /&gt;
 R/Peptides &lt;br /&gt;
 GOCC: Soybase &lt;br /&gt;
 SignalP 5.0 &lt;br /&gt;
 Phyre2

---

## A large deletion removed the CAP domain of a *GmPR-1*

.pull-left-2[
&lt;img src="figs/pr1/msa.png" width="1387" /&gt;
]

.pull-right-1[
&lt;img src="figs/pr1/3d.png" width="1581" /&gt;

.bgg[Method:] &lt;br /&gt;
MAFFT

]

---

## A large deletion removed the CAP domain of a *GmPR-1*

.pull-left-1[
&lt;img src="figs/pr1/msa.png" width="1387" /&gt;

.bgg[Method:] &lt;br /&gt;
Phyre2

]

.pull-right-2[
&lt;img src="figs/pr1/3d.png" width="1581" /&gt;
]

---

## Recent local duplications shaped GmPR-1 expansion

&lt;img src="figs/pr1/fig2.png" width="60%" style="display: block; margin: auto;" /&gt;

---

.pull-left-1[
2 phylogenetic groups.
]

.pull-right-2[
&lt;img src="figs/pr1/fig3A.png" width="1319" /&gt;
]

---

.pull-left-1[
2 phylogenetic groups.

3 genes with 2+ exons (PR-1 genes in other species are intronless)


]

.pull-right-2[
&lt;img src="figs/pr1/fig3B.png" width="1319" /&gt;
]

---

.pull-left-1[
2 phylogenetic groups.

3 genes with 2+ exons (PR-1 genes in other species are intronless)

Number of  genes and motifs reflect phylogeny.

]

.pull-right-2[
&lt;img src="figs/pr1/fig3C.png" width="1319" /&gt;
]

---

.pull-left-1[
2 phylogenetic groups.

3 genes with 2+ exons (PR-1 genes in other species are intronless)

Number of  genes and motifs reflect phylogeny.

GmPR-1 duplicates under strong purifying selection.

- 48% DD 
- 24% WGD 
- 12% TRD 
- 10% PD 
- 7% TD
]

.pull-right-2[
&lt;img src="figs/pr1/fig3D.png" width="1319" /&gt;
]

---

.pull-left-1[
2 phylogenetic groups.

3 genes with 2+ exons (PR-1 genes in other species are intronless)

Number of  genes and motifs reflect phylogeny.

GmPR-1 duplicates under strong purifying selection.

- 48% DD 
- 24% WGD 
- 12% TRD 
- 10% PD 
- 7% TD


Most abundant CREs are associated with stresses (C2H2, WRKY, NAC, GATA, ERF).
]

.pull-right-2[
&lt;img src="figs/pr1/fig3.png" width="1319" /&gt;
]

---

## Ancient WGD contributed to *GmPR-1* gene expansion

&lt;br /&gt;

Whole-genome duplications are the second main mechanism for *GmPR-1* expansion.


$$
DT (mya) = \frac{Ks}{2 \times 6.1e-09 \times 1e06}
$$

Most WGD-derived genes diverged in the 58 mya WGD event or in even more ancient events. 

---

## Most *GmPR-1* genes are PR-like proteins

&lt;br /&gt;

&lt;img src="figs/pr1/global.png" width="1828" /&gt;

23 of 24 *GmPR-1* genes are also expressed in healthy conditions (PR-like, probable endogenous function).

---

## *GmPR-1* gene expression tends to be tissue-specific

.pull-left[
&lt;br /&gt;
`\(\tau\)` index of tissue specificity:

- `\(\tau\)` close to 1: tissue-specific profile.
- `\(\tau\)` close to .n[0]: broad expression profile.

`\(\tau\)` indices are closer to 1.
]

.pull-right[
&lt;img src="figs/pr1/violin.png" width="100%" /&gt;
]

---

## *GmPR-1* expression is modulated by abiotic stress

&lt;img src="figs/pr1/abiotic.png" width="1679" /&gt;

86% (21/24) of the *GmPR-1* genes are up-regulated under at least one
abiotic stress condition, such as:
- alkalinity, salt, cadmium, drought, heat, herbicide, high ozone levels, and Fe and P deficiency.

---

## *GmPR-1* expression is modulated by biotic stress

&lt;img src="figs/pr1/biotic.png" width="1692" /&gt;

.n[20/24] GmPR-1 genes are up-regulated by at least one biotic stress, such as:

- fungi (*Fusarium oxysporum*, *Fusarium virguliforme*, *Cadophora gregata*, and *Phakopsora pachyrhizi*), insect (*Aphis glycines*), nematodes (*Heterodera glycines*, *Rotylenchulus reniformis*), oomycete (*Phytophthora sojae*) and virus (*soybean mosaic virus*).

---

## *GmPR-1* expression is modulated by biotic stress

&lt;img src="figs/pr1/biotic.png" width="1692" /&gt;

*Phytophthora sojae* and *Fusarium virguliforme* systematically repressed the expression of GmPR-1 genes.

GmPR-1-1, GmPR-1-16 and GmPR-1-18 were upregulated in the greatest number of biotic stresses, and GmPR-1-7, GmPR-1-21, GmPR-1-22 in abiotic stresses &amp;rarr; potential targets for biotechnological applications.

---

## Conclusions

&lt;br /&gt;
- Soybean genome has a PR-1 repertoire of 24 genes

--

- There are 2 phylogenetically distinct *GmPR-1* groups.

--

- Protein structures (primary and tertiary) are highly conserved, except *GmPR-1-14*.

--

- Evolution of *GmPR-1* genes is constrained by strong purifying selection.

--

- Major stress-related TF families regulate the expression of *GmPR-1* genes (WRKY, NAC, C2H2, ERF)

--

- Overall, *GmPR-1* genes are responsive to a myriad of biotic and abiotic stresses.

--

- We pinpointed 6 genes that are promising targets for biotechnological applications.

---

## Interested in more details?

&lt;br /&gt;
&lt;img src="figs/pr1/preprint.png" width="80%" style="display: block; margin: auto;" /&gt;

.center[.n[https://www.biorxiv.org/content/10.1101/2021.03.27.437342v1]]

---

background-image: url(https://secure-ecsd.elsevier.com/covers/80/Tango2/large/25901583.jpg)
background-position: 85% 50%
background-size: 25%

## Interested in more details?

&lt;br /&gt;
.font120[
Recently accepted in Gene. 

Final version will be available soon.
]

---

class: sydney-yellow, middle, center

## You can find me at:

&lt;svg viewBox="0 0 512 512" style="height:1em;position:relative;display:inline-block;top:.1em;" xmlns="http://www.w3.org/2000/svg"&gt;  &lt;path d="M464 64H48C21.49 64 0 85.49 0 112v288c0 26.51 21.49 48 48 48h416c26.51 0 48-21.49 48-48V112c0-26.51-21.49-48-48-48zm0 48v40.805c-22.422 18.259-58.168 46.651-134.587 106.49-16.841 13.247-50.201 45.072-73.413 44.701-23.208.375-56.579-31.459-73.413-44.701C106.18 199.465 70.425 171.067 48 152.805V112h416zM48 400V214.398c22.914 18.251 55.409 43.862 104.938 82.646 21.857 17.205 60.134 55.186 103.062 54.955 42.717.231 80.509-37.199 103.053-54.947 49.528-38.783 82.032-64.401 104.947-82.653V400H48z"&gt;&lt;/path&gt;&lt;/svg&gt;  [fabricio_almeidasilva@hotmail.com](mailto:fabricio_almeidasilva@hotmail.com)

&lt;svg viewBox="0 0 496 512" style="height:1em;position:relative;display:inline-block;top:.1em;" xmlns="http://www.w3.org/2000/svg"&gt;  &lt;path d="M336.5 160C322 70.7 287.8 8 248 8s-74 62.7-88.5 152h177zM152 256c0 22.2 1.2 43.5 3.3 64h185.3c2.1-20.5 3.3-41.8 3.3-64s-1.2-43.5-3.3-64H155.3c-2.1 20.5-3.3 41.8-3.3 64zm324.7-96c-28.6-67.9-86.5-120.4-158-141.6 24.4 33.8 41.2 84.7 50 141.6h108zM177.2 18.4C105.8 39.6 47.8 92.1 19.3 160h108c8.7-56.9 25.5-107.8 49.9-141.6zM487.4 192H372.7c2.1 21 3.3 42.5 3.3 64s-1.2 43-3.3 64h114.6c5.5-20.5 8.6-41.8 8.6-64s-3.1-43.5-8.5-64zM120 256c0-21.5 1.2-43 3.3-64H8.6C3.2 212.5 0 233.8 0 256s3.2 43.5 8.6 64h114.6c-2-21-3.2-42.5-3.2-64zm39.5 96c14.5 89.3 48.7 152 88.5 152s74-62.7 88.5-152h-177zm159.3 141.6c71.4-21.2 129.4-73.7 158-141.6h-108c-8.8 56.9-25.6 107.8-50 141.6zM19.3 352c28.6 67.9 86.5 120.4 158 141.6-24.4-33.8-41.2-84.7-50-141.6h-108z"&gt;&lt;/path&gt;&lt;/svg&gt;  [almeidasilvaf.github.io](https://almeidasilvaf.github.io/)


&lt;svg viewBox="0 0 512 512" style="height:1em;position:relative;display:inline-block;top:.1em;" xmlns="http://www.w3.org/2000/svg"&gt;  &lt;path d="M459.37 151.716c.325 4.548.325 9.097.325 13.645 0 138.72-105.583 298.558-298.558 298.558-59.452 0-114.68-17.219-161.137-47.106 8.447.974 16.568 1.299 25.34 1.299 49.055 0 94.213-16.568 130.274-44.832-46.132-.975-84.792-31.188-98.112-72.772 6.498.974 12.995 1.624 19.818 1.624 9.421 0 18.843-1.3 27.614-3.573-48.081-9.747-84.143-51.98-84.143-102.985v-1.299c13.969 7.797 30.214 12.67 47.431 13.319-28.264-18.843-46.781-51.005-46.781-87.391 0-19.492 5.197-37.36 14.294-52.954 51.655 63.675 129.3 105.258 216.365 109.807-1.624-7.797-2.599-15.918-2.599-24.04 0-57.828 46.782-104.934 104.934-104.934 30.213 0 57.502 12.67 76.67 33.137 23.715-4.548 46.456-13.32 66.599-25.34-7.798 24.366-24.366 44.833-46.132 57.827 21.117-2.273 41.584-8.122 60.426-16.243-14.292 20.791-32.161 39.308-52.628 54.253z"&gt;&lt;/path&gt;&lt;/svg&gt;  [@almeidasilvaf](https://twitter.com/almeidasilvaf)


&lt;svg viewBox="0 0 496 512" style="height:1em;position:relative;display:inline-block;top:.1em;" xmlns="http://www.w3.org/2000/svg"&gt;  &lt;path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"&gt;&lt;/path&gt;&lt;/svg&gt;  [almeidasilvaf](https://github.com/almeidasilvaf/)

    </textarea>
<style data-target="print-only">@media screen {.remark-slide-container{display:block;}.remark-slide-scaler{box-shadow:none;}}</style>
<script src="https://remarkjs.com/downloads/remark-latest.min.js"></script>
<script src="https://platform.twitter.com/widgets.js"></script>
<script>var slideshow = remark.create({
"ratio": "18:10",
"highlightStyle": "github",
"highlightLines": true,
"countIncrementalSlides": false,
"navigation": {
"scroll": false
}
});
if (window.HTMLWidgets) slideshow.on('afterShowSlide', function (slide) {
  window.dispatchEvent(new Event('resize'));
});
(function(d) {
  var s = d.createElement("style"), r = d.querySelector(".remark-slide-scaler");
  if (!r) return;
  s.type = "text/css"; s.innerHTML = "@page {size: " + r.style.width + " " + r.style.height +"; }";
  d.head.appendChild(s);
})(document);

(function(d) {
  var el = d.getElementsByClassName("remark-slides-area");
  if (!el) return;
  var slide, slides = slideshow.getSlides(), els = el[0].children;
  for (var i = 1; i < slides.length; i++) {
    slide = slides[i];
    if (slide.properties.continued === "true" || slide.properties.count === "false") {
      els[i - 1].className += ' has-continuation';
    }
  }
  var s = d.createElement("style");
  s.type = "text/css"; s.innerHTML = "@media print { .has-continuation { display: none; } }";
  d.head.appendChild(s);
})(document);
// delete the temporary CSS (for displaying all slides initially) when the user
// starts to view slides
(function() {
  var deleted = false;
  slideshow.on('beforeShowSlide', function(slide) {
    if (deleted) return;
    var sheets = document.styleSheets, node;
    for (var i = 0; i < sheets.length; i++) {
      node = sheets[i].ownerNode;
      if (node.dataset["target"] !== "print-only") continue;
      node.parentNode.removeChild(node);
    }
    deleted = true;
  });
})();
(function() {
  "use strict"
  // Replace <script> tags in slides area to make them executable
  var scripts = document.querySelectorAll(
    '.remark-slides-area .remark-slide-container script'
  );
  if (!scripts.length) return;
  for (var i = 0; i < scripts.length; i++) {
    var s = document.createElement('script');
    var code = document.createTextNode(scripts[i].textContent);
    s.appendChild(code);
    var scriptAttrs = scripts[i].attributes;
    for (var j = 0; j < scriptAttrs.length; j++) {
      s.setAttribute(scriptAttrs[j].name, scriptAttrs[j].value);
    }
    scripts[i].parentElement.replaceChild(s, scripts[i]);
  }
})();
(function() {
  var links = document.getElementsByTagName('a');
  for (var i = 0; i < links.length; i++) {
    if (/^(https?:)?\/\//.test(links[i].getAttribute('href'))) {
      links[i].target = '_blank';
    }
  }
})();
// adds .remark-code-has-line-highlighted class to <pre> parent elements
// of code chunks containing highlighted lines with class .remark-code-line-highlighted
(function(d) {
  const hlines = d.querySelectorAll('.remark-code-line-highlighted');
  const preParents = [];
  const findPreParent = function(line, p = 0) {
    if (p > 1) return null; // traverse up no further than grandparent
    const el = line.parentElement;
    return el.tagName === "PRE" ? el : findPreParent(el, ++p);
  };

  for (let line of hlines) {
    let pre = findPreParent(line);
    if (pre && !preParents.includes(pre)) preParents.push(pre);
  }
  preParents.forEach(p => p.classList.add("remark-code-has-line-highlighted"));
})(document);</script>

<script>
slideshow._releaseMath = function(el) {
  var i, text, code, codes = el.getElementsByTagName('code');
  for (i = 0; i < codes.length;) {
    code = codes[i];
    if (code.parentNode.tagName !== 'PRE' && code.childElementCount === 0) {
      text = code.textContent;
      if (/^\\\((.|\s)+\\\)$/.test(text) || /^\\\[(.|\s)+\\\]$/.test(text) ||
          /^\$\$(.|\s)+\$\$$/.test(text) ||
          /^\\begin\{([^}]+)\}(.|\s)+\\end\{[^}]+\}$/.test(text)) {
        code.outerHTML = code.innerHTML;  // remove <code></code>
        continue;
      }
    }
    i++;
  }
};
slideshow._releaseMath(document);
</script>
<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
(function () {
  var script = document.createElement('script');
  script.type = 'text/javascript';
  script.src  = 'https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML';
  if (location.protocol !== 'file:' && /^https?:/.test(script.src))
    script.src  = script.src.replace(/^https?:/, '');
  document.getElementsByTagName('head')[0].appendChild(script);
})();
</script>
  </body>
</html>
